### DISCUSSION TITLE
寻求回到侧面笔记预览老界面的方法

### DISCUSSION THREAD

**Discussion by spikelovejulia** [MuiseDestiny/zotero-style#468] [Q&A] [answered]
做笔记后想加个#标签，之后突然发现侧面笔记预览和以前不一样了，现在单击条目是这样子：
![image](https://github.com/MuiseDestiny/zotero-style/assets/51293923/1ea774f2-ff9f-4468-bd98-63c865c1165c)
我看不到笔记的图😭😰，如果要看到图的话必须展开条目直接点击笔记附件预览：
![image](https://github.com/MuiseDestiny/zotero-style/assets/51293923/291c01d3-3a30-44ed-9b52-56dcd4ec2dc4)
怎么变回去啊，我不需要这个新的笔记预览界面，我很难受

**Comment by spikelovejulia** [comment]
帮帮孩子😥😥😥

**Reply by MuiseDestiny** [reply]
看下首选项可不可以关闭
![image](https://github.com/MuiseDestiny/zotero-style/assets/51939531/9078b674-efa3-4714-b5ab-51ed0b422f4c)

**Reply by spikelovejulia** [reply]
谢谢，我没有找到render item annotation这个按钮，不过我已经明白了，还有我想知道
![image](https://github.com/MuiseDestiny/zotero-style/assets/51293923/13a46433-4058-4e27-8117-185481ec07b0)
这个的file应该是保存style样式吗？这样子应该可以跨设备同步style设置吧，默认的note代表的是文库里面那个addon item？😮

**Comment by MuiseDestiny** [comment, marked as answer]
预览笔记需要点击笔记。点击条目只会显示她下面的笔记条目。不会直接预览。

**Reply by spikelovejulia** [reply]
好像是这么回事，我之前没有仔细关注，谢谢！
